xcp_d/utils/bids.py

```diff
@@ -264,6 +272,9 @@ def collect_data(
         )["nifti"]
         for space in temp_allowed_spaces:
             temp_bold_query["space"] = space
+            if input_type == "hcp" or input_type == "dcan":
```

Comment:
```suggestion
            if input_type in ("hcp", "dcan"):
```

Reply:
Can you add a comment about why this is done? I assume it's because there's no preprocessed BOLD data in nifti format, and you're using the boldref instead?

Also, I think you can move this if statement to outside the for loop, since it's looping through spaces, and `desc` and `suffix` should stay the same across space-specific queries.